mwcites/extractors/isbn.py

```diff
@@ -4,7 +4,8 @@
 # Also correctly parses malformed inputs such as below:
 # isbn=2 906700-09-6 (notice the space instead of a hyphen) or
 # isbn=2 10 004179 7 (notice spaces instead of hyphens)
-ISBN_RE = re.compile('isbn\s?=?\s?([\d]+([\d\s\-]+)[\dXx])', re.I)
+# {{ISBN|978-83-7435-239-0​}} (notice pipe instead of equals)
+ISBN_RE = re.compile('isbn\s?[=|\|]?\s?([\d]+([\d\s\-]+)[\dXx])', re.I)
```

Comment:
I don't think you need the first pipe here, do you? Brackets already mean that either `=` or `\` will be matched.